src/app/components/navbar/navbar.component.css

```
@@ -0,0 +1,26 @@
+.navbar{
+  background: #ffffff;
+}
+
+.grid-navbar-item {
+  width: 80%;
+  display: flex;
+  margin: auto;
+  justify-content: space-between;
+}
+
+.title {
+  color: #7a8dfd;
+  font-size: 36px;
+}
+
+img.titleLogo{
```

```suggestion
img.title-logo {
```

Traditional convention for class name is `kebab-case` and variable name is `camelCase` in JS.